src/ocrd_utils/config.py

```diff
@@ -141,6 +143,30 @@ def _ocrd_download_timeout_parser(val):
     description="Timeout in seconds for connecting or reading (comma-separated) when downloading.",
     parser=_ocrd_download_timeout_parser)
 
+config.add("OCRD_DOWNLOAD_INPUT",
+    description="Whether to download files not present locally during processing",
+    default=(True, True),
+    validator=_validator_boolean,
+    parser=_parser_boolean)
+
+config.add("OCRD_MISSING_INPUT",
+    description="How to deal with missing input files (for some fileGrp/pageId) during processing [SKIP|ABORT]",
+    default=(True, 'SKIP'),
+    validator=lambda val: val in ['SKIP', 'ABORT'],
+    parser=str)
+
+config.add("OCRD_MISSING_OUTPUT",
+    description="How to deal with missing output files (for some fileGrp/pageId) during processing [SKIP|COPY|ABORT]",
+    default=(True, 'SKIP'),
+    validator=lambda val: val in ['SKIP', 'COPY', 'ABORT'],
+    parser=str)
+
+config.add("OCRD_EXISTING_OUTPUT",
+    description="How to deal with already existing output files (for some fileGrp/pageId) during processing [SKIP|OVERWRITE|ABORT]",
+    default=(True, 'SKIP'),
+    validator=lambda val: val in ['SKIP', 'OVERWRITE', 'ABORT'],
+    parser=str)
+
```

Comment:
It is not urgent, but I would also appreciate a more detailed description of each option. What happens when the output is missing and a COPY option is selected? What is being copied? It was maybe discussed in more detail somewhere but I have missed it. The existing output options are straightforward.

Reply:
The comment was supposed to go for the `OCRD_MISSING_OUTPUT` context.

Reply:
The discussion is here: https://github.com/bertsky/core/commit/f00ecda84717d274126a736b67e3ab29e5bae83d#r145600564